src/app/components/sites/pages/wizard/wizard.component.ts

```diff
@@ -86,6 +86,7 @@ WizardComponent.linkToRoute({
   pageRoute: newSiteMenuItem,
   menus: { actions: List(projectMenuItemActions) },
   resolvers: { [projectKey]: projectResolvers.show },
+  title: () => "New Region / Site",
```

Comment:
```suggestion
  title: () => "New Site or Point",
```

Remember, the term _region_ is a technical concept - it should be exposed nowhere in the UI.